src/bin/pg_autoctl/pgsql.c

```diff
@@ -1553,35 +1576,35 @@ pgsql_replication_slot_create_and_drop(PGSQL *pgsql, NodeAddressArray *nodeArray
 	char values[BUFSIZE] = { 0 };
 
 	/* *INDENT-OFF* */
-	char *sqlTemplate =
-		/*
-		 * We could simplify the writing of this query, but we prefer that it
-		 * looks as much as possible like the query used in
-		 * pgsql_replication_slot_maintain() so that we can maintain both
-		 * easily.
-		 */
-		"WITH nodes(slot_name, lsn) as ("
-		" SELECT '" REPLICATION_SLOT_NAME_DEFAULT "_' || id, lsn"
-		"   FROM (%s) as sb(id, lsn) "
-		"), \n"
-		"dropped as ("
-		" SELECT slot_name, pg_drop_replication_slot(slot_name) "
-		"   FROM pg_replication_slots pgrs LEFT JOIN nodes USING(slot_name) "
-		"  WHERE nodes.slot_name IS NULL "
-		"    AND (   slot_name ~ '" REPLICATION_SLOT_NAME_PATTERN "' "
-		"         OR slot_name ~ '" REPLICATION_SLOT_NAME_DEFAULT "' )"
-		"    AND not active"
-		"    AND slot_type = 'physical'"
-		"), \n"
-		"created as ("
-		"SELECT c.slot_name, c.lsn "
-		"  FROM nodes LEFT JOIN pg_replication_slots pgrs USING(slot_name), "
-		"       LATERAL pg_create_physical_replication_slot(slot_name, true) c"
-		" WHERE pgrs.slot_name IS NULL "
-		") \n"
-		"SELECT 'create', slot_name, lsn FROM created "
-		" union all "
-		"SELECT 'drop', slot_name, NULL::pg_lsn FROM dropped";
+    char *sqlTemplate =
+            /*
+             * We could simplify the writing of this query, but we prefer that it
+             * looks as much as possible like the query used in
+             * pgsql_replication_slot_maintain() so that we can maintain both
+             * easily.
+             */
+            "WITH nodes(slot_name, lsn) as ("
+            " SELECT '" REPLICATION_SLOT_NAME_DEFAULT "_' || id, lsn"
+            "   FROM (%s) as sb(id, lsn) "
+            "), \n"
+            "dropped as ("
+            " SELECT slot_name, pg_drop_replication_slot(slot_name) "
+            "   FROM pg_replication_slots pgrs LEFT JOIN nodes USING(slot_name) "
+            "  WHERE nodes.slot_name IS NULL "
+            "    AND (   slot_name ~ '" REPLICATION_SLOT_NAME_PATTERN "' "
+            "         OR slot_name ~ '" REPLICATION_SLOT_NAME_DEFAULT "' )"
+            "    AND not active"
+            "    AND slot_type = 'physical'"
+            "), \n"
+            "created as ("
+            "SELECT c.slot_name, c.lsn "
+            "  FROM nodes LEFT JOIN pg_replication_slots pgrs USING(slot_name), "
+            "       LATERAL pg_create_physical_replication_slot(slot_name, true) c"
+            " WHERE pgrs.slot_name IS NULL "
+            ") \n"
+            "SELECT 'create', slot_name, lsn FROM created "
+            " union all "
+            "SELECT 'drop', slot_name, NULL::pg_lsn FROM dropped";
```

Comment:
This looks like a whitespace change that we should not have in this PR, please fix.